create_bls_key.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/expect
 # 6 num wanted
-set wallet_password 123456 
+set wallet_password 12345678 
```

Comment:
let wallet_password be same as password for consensus addr
so can use the same password file in config